frontend/src/bundles/resume-access/store/slice.ts

```diff
@@ -0,0 +1,55 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import { type ResumeShareAccessGetResponseDto } from '../types/types.js';
+import {
+    accessResume,
+    accessResumeDetails,
+    deleteAccessResume,
+} from './actions.js';
+
+type State = {
+    resumeId: string | null;
+    details: ResumeShareAccessGetResponseDto[];
+};
+
+const initialState: State = {
+    resumeId: null,
+    details: [],
+};
+
+const { reducer, actions, name } = createSlice({
+    initialState,
+    name: 'resumeAccess',
+    reducers: {},
+    extraReducers(builder) {
+        builder.addCase(accessResume.pending, (state) => {
+            state.resumeId = null;
+        });
+
+        builder.addCase(accessResume.fulfilled, (state, action) => {
+            state.resumeId = action.payload.resumeId;
+        });
+
+        builder.addCase(accessResume.rejected, (state) => {
+            state.resumeId = null;
+        });
+
+        builder.addCase(deleteAccessResume.fulfilled, (state) => {
+            state.resumeId = null;
+        });
+
```

Comment:
can't addMatcher be used here? for `deleteAccessResume` and `accessResume` - for sure, try to use matcher for all cases together